txpool/grpc_server.go

```diff
@@ -264,11 +264,17 @@ func (s *GrpcServer) Transactions(ctx context.Context, in *txpool_proto.Transact
 
 	reply := &txpool_proto.TransactionsReply{RlpTxs: make([][]byte, len(in.Hashes))}
 	for i := range in.Hashes {
-		h := gointerfaces.ConvertH256ToHash(in.Hashes[i])
-		txnRlp, err := s.txPool.GetRlp(tx, h[:])
-		if err != nil {
-			return nil, err
+		var h [32]byte
+		var txnRlp []byte
+
+		if in.Hashes[i] != nil {
```

Comment:
I think the issue is not that `in.Hashes[i]` is `nil`, but because `txnRlp` is `nil`